src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -59,6 +59,7 @@
   private static final String DEV_APPSERVER_PY = "bin/dev_appserver.py";
   private static final String JAVA_APPENGINE_SDK_PATH =
       "platform/google_appengine/google/appengine/tools/java/lib";
+  private static final String JAVA_COMPONENTS_PATH = "platform/gcd/.appengine";
```

Comment:
@ludoch who would know when staging is going to make it to maven central.

To summarize what's being discussed:

For component detection we should be calling gcloud components list and not digging into the cloud SDK packaged files. 

The staging question is not related to this CL but we are working on moving as many deps as possible away from using Cloud SDK paths.